static_src/components/users.jsx

```diff
@@ -24,6 +24,14 @@ const ORG_MANAGER = 'org_manager';
 const SPACE_MANAGER = 'space_manager';
 const ORG_ENTITY = 'organization';
 const SPACE_ENTITY = 'space';
+const ORG_INVITE_HELP = 'Only an Org Manager can new invite users to this ' +
```

Comment:
agreed. there's a constants.js file too we should probably move things there eventually but should make we group them accordingly.